pkg/services/extsvcauth/extsvcaccounts/service.go

```diff
@@ -16,6 +16,10 @@ import (
 	sa "github.com/grafana/grafana/pkg/services/serviceaccounts"
 )
 
+const (
+	extsvcPrefix = "extsvc"
```

Comment:
I'd suggest placing it here: https://github.com/grafana/grafana/blob/7d99da377fe8ce537b3c4c88dba50e4b820288ce/pkg/services/extsvcauth/extsvcaccounts/models.go#L11

Reply:
Changed